src/repos/baseRepo.js

```diff
@@ -0,0 +1,44 @@
+const { EntityIsNotDefinedError, NotFound } = require('./common')
+
+class BaseRepo {
+    static get entity() {
+        throw new EntityIsNotDefinedError()
+    }
+
+    static get table() {
+        throw new TableIsNotDefinedError()
+    }
+
+    constructor(gateway, deps = {}) {
+        this.gateway = gateway
+        this.deps = deps
```

Comment:
Do you just access these properties here or do you create them through the method arguments?

Reply:
This properties are passed in constructor. 
https://github.com/sychova/nodeJS_course-chat_app/blob/6effd77dac76569824def205b451b2f8b46080b9/src/repos/index.js#L5

Reply:
I'm sorry but this is probably not what I mean.
This is pretty much the same issue I described above here:
https://github.com/sychova/nodeJS_course-chat_app/pull/2#discussion_r776573412

To rephrase it:
Do these blue "gateway"/"deps" propeties of THIS get created from the names of the properties from the constructor?
(I suppose not)
Do they "depend" on them?

Reply:
no
no